tradeexecutor/ethereum/constants.py

```diff
@@ -0,0 +1,7 @@
+"""Defines constants for routing, especially fee data for different exchanges"""
```

Comment:
These are no longer needed, as this has been fixed in the backend  and fees are part of DEXPair dataset.